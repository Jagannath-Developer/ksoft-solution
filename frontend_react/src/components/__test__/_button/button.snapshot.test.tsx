import renderer from "react-test-renderer";
import { Button } from "../..";

describe("Jest Snapshot testing suite", () => {
  it("Matches DOM Snapshot", () => {
    const domTree = renderer
      .create(
        <Button
          onClick={function (): void {
            throw new Error("Function not implemented.");
          }}
        />
      )
      .toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
