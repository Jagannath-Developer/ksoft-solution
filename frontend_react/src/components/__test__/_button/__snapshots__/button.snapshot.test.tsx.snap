// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<button
  className="ant-btn css-dev-only-do-not-override-1rqnfsa ant-btn-default"
  disabled={false}
  onClick={[Function]}
  style={Object {}}
  type="button"
>
  <span>
    first test
  </span>
</button>
`;
